packages/design-system/src/components/snackbar/snackbarContainer.js

```diff
@@ -170,6 +171,7 @@ export const SnackbarContainer = ({
               <ChildContainer ref={nodeRef} placement={placement}>
                 <Component
                   {...notificationProps}
+                  aria-label={ariaLabel}
```

Comment:
That does not seem right. `aria-label` should already be part of `notificationProps`.

This is the aria label for the individual message, not for the container.

So adding a label like "Snackbar for dashboard" would be incorrect.

Reply:
Oh my bad but after further research i found out that `showSnackbar` only sends down only these 3 things in object
```
dismissible: true
id: "7e9ce02c-8d17-46c5-9d05-7f9cf81346aa"
message: "Message to be shown in snackbar"
```
so would it be nice if we send `aria-label` for the snackbar something like this `'aria-label': 'Notification for '+message,` or `'aria-label': 'Snackbar for '+message,`